action.yml

```diff
@@ -8,6 +8,10 @@ inputs:
   tool_versions:
     required: false
     description: If present, this value will be written to the .tool-versions file
+  install:
+    required: false
+    default: 'true'
```

Comment:
Shouldn't this just be `true`?

Reply:
According to the GitHub document, the value should be `string` .
https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault

Reply:
I changed this to `true`.